app/controllers/patients_controller.rb

```diff
@@ -0,0 +1,66 @@
+class PatientsController < ApplicationController
+  def index
+    @can_create = has_access?(@navigation_context.institution, CREATE_INSTITUTION_PATIENT)
+    @patients = check_access(Patient.where(institution: @navigation_context.institution), READ_PATIENT).order(updated_at: :desc)
+  end
```

Comment:
If patients don't have a site, then the only way for a user to browse patients is to have permissions over the whole institution. Should we add something more related to site?